templates/deployment.yml

```diff
@@ -28,9 +26,9 @@ spec:
       - name: {{ .Values.imagePullSecrets }}
       {{- end }}
       containers:
-      - image: armory/kubesvc:0.6.0-rc.40
+      - image: armory/agent-k8s:{{ .appVersion }}
```

Comment:
I noticed that this wasn't using appVersion before and I was curious if that was intentional?

Reply:
I don't think it was intentional, it's good to use appVersion here